docs/network/wwivnet_linux.md

```diff
@@ -103,105 +51,112 @@ dosemu -dumb -quiet "$(basename ${0}).bat ${NET}" 2>/dev/null
+ 
+ `network3 y .0`
+ 
+ If you leave out the network number, .0 is assumed.
 
-(number of target) = the number of the system you get/send messages from/to (typically 1)
+* `network3` will send you an email to verify your setup. In order for the BBS 
+to pick it up, you should run `bbs -M`.  **NOTE** While the local node defaults to
+running with -M, it will *only* pick the mail up if the local node is started 
+after you have run `network3`.
 
+* Login as #1 and Check your email on the BBS. You should see a pretty email
+  and it should say "Everything is Fine" along with other details.
 
-So, a command for a typical setup will look like this:
 
-```network --network=wwivnet --node=1```
+## Verify Network Mail
 
-Full networkb usage looks like:
-```
-./network [args]
+* Login to your BBS and send an email to `1@1`
+* You will be prompted to pick the net (WWVIVnet) and confirm the system (Weather Station).
+* Give it a subject. Send the Net Coordinator a love note and hit /S to save and send.
+* If you and 1@1 are chatting in IRC, you can confirm and troubleshoot all at once. 
+Otherwise, please give 1@1 at least several hours to read and reply.  
 
-program arguments:
---allow_sendback       Allow sendback (only used by legacy network0)
---bbsdir               (optional) BBS directory if other than current directory
---callout_time         Start time of the callout (only used by legacy network0)
---help                 displays help.
---network              Network name to use (i.e. wwivnet).
---network_number       Network number to use (i.e. 0).
---node                 Network node number to dial.
---phone_number         Network number to use (only used by legacy network0)
---speed                Modem Speedto use (only used by legacy network0)
-/N####     Network node number to dial.
-.####      Network number (as defined in wwivconfig)
-```
+## Mail Processing
 
+Network mail processing **must** be done from ${WWIV_DIR}.  The basic command 
+to invoke message transport is:
 
-Full networkb (the command that network calls to do the work) usage looks like:
+`network --net=(name or position of your network) --node=(number of target)`
 
-```
-Usage: networkb [flags]
-Flags:
---network  Network name to use (i.e. wwivnet)
---bbsdir   (optional) BBS directory if other than current directory 
---send     Send network traffic to --node
---receive  Receive from any node
---node     Node number (only used when sending)
---port     Port number to use (receiving only)
---skip_net Skip invoking network1/network2/network3
-```
+So to continue with the example above, it could be:
 
-If you want to allow your system to listen for incoming connections to BinkP, that's possible to do with the --receive flag; but it's a one-shot service, so it needs to be put in a loop (more details to come on this).  Most people will be perfectly fine with just running the network command on a schedule.
+`network --net=wwivnet --node=1`
```

Comment:
the .0 works too.  I just do ```network -n1```

Reply:
added

Reply:
Note - not *committed* yet, lol.  Finishing these on breaks while I"m at work.